.drone.yml

```diff
@@ -3113,9 +3113,9 @@ steps:
     ARCH: amd64
     BUILDBOX_PASSWORD:
       from_secret: BUILDBOX_PASSWORD
-    ENT_TARBALL_PATH: /tmp/build-darwin-amd64-pkg-tsh/go/artifacts
+    ENT_TARBALL_PATH: /tmp/build-darwin-amd64-pkg/go/artifacts
     OS: darwin
-    OSS_TARBALL_PATH: /tmp/build-darwin-amd64-pkg-tsh/go/artifacts
+    OSS_TARBALL_PATH: /tmp/build-darwin-amd64-pkg/go/artifacts
```

Comment:
These changes should not have been made. You might need to rebase, here's the PR where I made the fix: https://github.com/gravitational/teleport/pull/10861.

Reply:
This has been fixed